web/packages/teleport/src/config.ts

```diff
@@ -515,10 +516,33 @@ const cfg = {
     });
   },
 
-  getScpUrl(params: UrlScpParams) {
-    return generatePath(cfg.api.scp, {
-      ...params,
+  getScpUrl({
+    webauthn,
+    login,
+    location,
+    filename,
+    clusterId,
+    serverId,
+  }: UrlScpParams) {
+    let path = generatePath(cfg.api.scp, {
+      login,
+      location,
+      filename,
+      clusterId,
+      serverId,
     });
```

Comment:
Nit:

```suggestion
  getScpUrl({ webauthn, ...pathParams }: UrlScpParams) {
    let path = generatePath(cfg.api.scp, pathParams);
```

You could also do `{ ...pathParams }` on the second line as it was before. Though I don't fully understand why it was done here.

Reply:
I just totally forgot about the spread here for deconstruction. Thanks for the reminder